pyproject.toml

```diff
@@ -103,6 +103,7 @@ exclude = '''
 ###########
 [tool.isort]
 known_first_party = ["versioneer"]
+known_third_party = ["nox"]
```

Comment:
i was having issues locally with `isort` picking up `nox` as a "first party" library (this was _not_ happening on CI). this forces `isort` to recognize `nox` as a _third party_ library in all contexts.